### PR TITLE
refactor secret in report and report path

### PR DESCRIPTION
**Pull request by momentmaker**

### CODE REVIEW

File: gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts [hunk 1]
```diff
@@ -231,6 +231,15 @@ export default class ProposeOffchainConfig extends SolanaCommand {
     const rawTx = await this.makeRawTransaction(this.wallet.publicKey)
     const startingPoint = new BN(this.flags.instruction || 0).toNumber()
 
+    const input = this.makeInput(this.flags.input)
```
Comment (momentmaker) [inline gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts:234]: If there is a better way to get `randomSecret` like from `responses` I can refactor

Comment (RodrigoAD) [reply gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts:234]: This won't work unfortunately, let me think on a better way

**Comment by momentmaker** [comment]
@RodrigoAD awesome! that should work :) could you please approve it?

**Review by RodrigoAD — APPROVED** [review]

**Comment by RodrigoAD** [comment]
can you rebase with develop @momentmaker ? CI seems to be failing

**Pull request merged**